cli.js

```diff
@@ -10,7 +10,9 @@ const cli = meow(`
 	  $ np [major | minor | patch | premajor | preminor | prepatch | prerelease | <version>] (Default: patch)
 
 	Options
-	  --any-branch  Allow publishing from any branch
+	  --any-branch    Allow publishing from any branch
+	  --skip-cleanup  Skips cleanup of node_modules
+	  --yolo          Skips cleanup of node_modules and tests
```

Comment:
Description isn't very clear. It looks like it skips the cleanup of `node_modules` and the cleanup of `tests`. Something like `Skips cleanup of node_modules and skips testing`.